manoseimas/mps_v2/static/components/filter_component.jsx

```diff
@@ -297,9 +298,9 @@ var ElementList = React.createClass({
     var self = this;
     return (
       <div className="filtered-elements">
-        {self.props.items.map(function (item) {
+        {self.props.items.map(function (item, index) {
           return (
-              <self.props.rowComponent obj={item}
+              <self.props.rowComponent obj={item} key={index}
```

Comment:
Using array index as a key here might cause the components to be reused instead of recreated. While this is not a problem in this particular case it might cause issues if using external stuff like jQuery plugins on the underlying DOM element. A unique key would be needed to handle that correctly.